package.json

```diff
@@ -34,19 +30,29 @@
   "license": "MIT",
   "scripts": {
     "build": "microbundle -i ./src/main.js -o lib",
-    "test": "node ./test/tester.js",
+    "test": "jest --forceExit --silent --coverage",
     "test:server": "node ./test/support/startServer.js",
     "watch": "microbundle watch -i ./src/main.js -o lib"
   },
+  "jest": {
+    "testEnvironment": "node",
+    "globalSetup": "<rootDir>/test/support/jest.globalsetup.js",
+    "setupTestFrameworkScriptFile": "<rootDir>/test/support/jest.setup.js",
+    "collectCoverageFrom": ["src/**/*.js"],
+    "verbose": true
+  },
   "dependencies": {
     "ansi-colors": "^1.1.0",
     "puppeteer": "^1.1.1",
     "yargs": "^11.0.0"
   },
   "devDependencies": {
     "ansi_up": "^2.0.2",
+    "babel-jest": "22.4.1",
```

Comment:
Do we need to add babel-core?

Reply:
babel-core is installed via other deps. we dont need to explicitly require it